<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee</class>
 <widget class="QDialog" name="Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Работники</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="emplsTableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Добавление работника</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="2" colspan="2">
       <widget class="QLineEdit" name="emplTabNum"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="emplName"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Должность</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="emplFirstName"/>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="emplSerName"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Отчество</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Табельный номер</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QTableView" name="positionsTableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Employee</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Employee</receiver>
   <slot>delEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Employee</receiver>
   <slot>addEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addEmployee()</slot>
  <slot>delEmployee()</slot>
 </slots>
</ui>
